test/rules/assertions/rejectAnyType.js

```diff
@@ -0,0 +1,42 @@
+export default {
+  invalid: [
```

Comment:
this misses test cases using the type as `*` instead of `any` (the rule implements this, but it is not tested)

Reply:
Done in `main`.